lib/armbian-configng/config.ng.jobs.json

```diff
@@ -75,6 +75,145 @@
+                                           "sed -i \"s/^#\\?PasswordAuthentication.*/PasswordAuthentication yes/\" /etc/ssh/sshd_config" ,
+                                           "systemctl restart sshd.service"
+                                       ],
+                            "status": "Active",
+                            "doc_link": "",
+                            "src_reference": "",
+                            "author": "",
+                            "condition": "grep -q 'PasswordAuthentication no' /etc/ssh/sshd_config"
+                            },
+                            {
+                            "id": "S12",
+                            "description": "Disable Public key authentication login",
+                            "command": [
+                                           "sed -i \"s/^#\\?PubkeyAuthentication.*/PubkeyAuthentication no/\" /etc/ssh/sshd_config" ,
+                                           "systemctl restart sshd.service"
+                                       ],
+                            "status": "Active",
+                            "doc_link": "",
+                            "src_reference": "",
+                            "author": "",
+                            "condition": "grep -q 'PubkeyAuthentication yes' /etc/ssh/sshd_config"
+                            },
+                            {
+                            "id": "S13",
+                            "description": "Enable Public key authentication login",
+                            "command": [
+                                           "sed -i \"s/^#\\?PubkeyAuthentication.*/PubkeyAuthentication yes/\" /etc/ssh/sshd_config" ,
+                                           "systemctl restart sshd.service"
+                                       ],
+                            "status": "Active",
+                            "doc_link": "",
+                            "src_reference": "",
+                            "author": "",
+                            "condition": "grep -q 'PubkeyAuthentication no' /etc/ssh/sshd_config"
+                            },
+                            {
+                            "id": "S14",
+                            "description": "Disable OTP authentication",
+                            "command": [
+                                           "clear",
+                                           "! check_if_installed libpam-google-authenticator && ! check_if_installed qrencode || debconf-apt-progress -- apt-get -y purge libpam-google-authenticator qrencode",
+                                           "sed -i \"s/^#\\?ChallengeResponseAuthentication.*/ChallengeResponseAuthentication no/\" /etc/ssh/sshd_config || sed -i \"0,/KbdInteractiveAuthentication/s//ChallengeResponseAuthentication yes/\" /etc/ssh/sshd_config" ,
+                                           "sed -i '/^auth required pam_google_authenticator.so nullok/ d' /etc/pam.d/sshd",
```

Comment:
Putting commands here will work, But will not be available though the --api feature 

example : try
```
sudo ./configng/bin/armbian-configng --api see_ping
``` 

```
sudo ./configng/bin/armbian-configng --api 
 check_ip_version
```